TheWarWithin/HunterBeastMastery.lua

```diff
@@ -1216,8 +1216,8 @@ spec:RegisterAura( "lethal_command", {
 
 
 local CallOfTheWildCDR = setfenv( function()
-    reduceCooldown( "kill_command", spec.abilities.kill_command.cooldown/2 )
-    reduceCooldown( "barbed_shot", spec.abilities.barbed_shot.cooldown/2 )
+    gainChargeTime( "kill_command", spec.abilities.kill_command.recharge_time/2)
```

Comment:
I think these should just be `.recharge` -- but I'm assuming this works in your testing.

Reply:
Yeah, there are spots in various APLs that use `recharge_time`, but I will use `recharge` to keep in line with other .lua files instead